drift/lib/src/runtime/query_builder/statements/insert.dart

```diff
@@ -214,6 +228,9 @@ class InsertStatement<T extends Table, D> {
 
     if (returning) {
       ctx.buffer.write(' RETURNING *');
+    } else if (ctx.dialect == SqlDialect.postgres) {
+      //TODO: handle multiple primary key
+      ctx.buffer.write(' RETURNING ${table.$primaryKey.first.$name}');
```

Comment:
If you don't want to add support for composite primary keys in this PR (which is fine with me), we should at least throw an exception explaining that they're not currently supported. Silently doing the wrong thing is not a good strategy :D

Reply:
Returning query isn't really necessary. This emulates SQLite's `last_insert_rowid` what is returns integer primary key. If there is no integer primary key or there is multiple key, SQLite return rowId. If table created `WITHOUT ROWID` then it returns zero. Follow this way, we could return zero if there is multiple keys or the primary key is not integer (non-integer primary key also not handled in this code)

Reply:
Fair enough, let's return an invalid value (say `-1`) if we hit an unexpected case. We should also document this current limitation somewhere, but I can take care of that.

Reply:
Having non-integer or multi key primary doesn't seem unexpected. I should return 0 as SQLite does.